### PR TITLE
Refactor code and comments from Person to Transaction, Address Book to Thrift

### PR DESCRIPTION
**Pull request by lightz96**
Todo🔜: 
1. The todo list mentioned in #62 
2. `JsonSerializableThriftTest#toModelType_typicalTransactionsFile_success`
3. `JsonSerializableThriftTest#toModelType_invalidTransactionFile_throwsIllegalValueException`
4. `JsonThriftStorageTest#treadAndSaveThrift_allInOrder_success`
5. `StorageManagerTest#thriftReadSave`

ℹ️ Remember to modify your preference.json to `"thriftFilePath" : "data\\thrift.json"` (or just delete the preference.json, config.json will regenerate it)

### CODE REVIEW

File: src/main/java/thrift/logic/commands/FindCommand.java [hunk 1]
```diff
@@ -7,15 +7,15 @@
...
+ * Finds and lists all transactions in THRIFT whose description contains any of the argument keywords.
  * Keyword matching is case insensitive.
  */
 public class FindCommand extends Command {
 
     public static final String COMMAND_WORD = "find";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
-            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all transaction whose description contain any "
```
Comment (lye-jw) [inline src/main/java/thrift/logic/commands/FindCommand.java]: Small grammar detail: "`Finds all transaction`**s**` whose description`**s**` contain any`"... as multiple transactions are found.

File: src/main/java/thrift/model/util/CurrencyUtil.java [hunk 2]
```diff
@@ -32,7 +32,7 @@
      * @return Amount in target currency as a double.
      */
     public static double convertFromDefaultCurrency(Map<String, Double> currencyMappings, double value,
-        String currency) {
+            String currency) {
```
Comment (lye-jw) [inline src/main/java/thrift/model/util/CurrencyUtil.java]: Does checkStyle complain that newline indent isn't 8 spaces? If not, ignore this comment.

File: src/main/java/thrift/storage/JsonAdaptedTransaction.java [hunk 3]
```diff
@@ -36,8 +36,8 @@
      */
     @JsonCreator
     public JsonAdaptedTransaction(@JsonProperty("type") String type, @JsonProperty("description") String description,
-                                  @JsonProperty("value") String value, @JsonProperty("date") String date,
-                                  @JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
+            @JsonProperty("value") String value, @JsonProperty("date") String date,
```
Comment (lye-jw) [inline src/main/java/thrift/storage/JsonAdaptedTransaction.java]: Does checkStyle complain that newline indents aren't 8 spaces? If not, ignore this comment.

**Review by lye-jw — COMMENTED** [review]
Great job refactoring so many instances. Just some minor observations on small details that may or may not require modifications. Will approve if nothing needs to be changed.

File: src/main/java/thrift/model/util/CurrencyUtil.java (hunk 2 above)
Comment (lightz96) [reply src/main/java/thrift/model/util/CurrencyUtil.java]: There are 8 spaces.

File: src/main/java/thrift/storage/JsonAdaptedTransaction.java (hunk 3 above)
Comment (lightz96) [reply src/main/java/thrift/storage/JsonAdaptedTransaction.java]: There are 8 spaces.

**Review by lye-jw — COMMENTED** [review]
LGTM!

**Review by lye-jw — APPROVED** [review]

**Pull request merged**